environments/prod/apps-hmcts-net.yml

```diff
@@ -8,4 +8,25 @@ txt: []
 
 srv: []
 
-cname: []
+cname: 
+  - name: "mcnotify"
+    ttl: 300
+    record: "hmcts-live-ext.azurefd.net"
+  - name: "mcol"
+    ttl: 300
+    record: "hmcts-live-ext.azurefd.net"
+  - name: "pcnotify"
+    ttl: 300
+    record: "hmcts-live-ext.azurefd.net"
+  - name: "pcol"
+    ttl: 300
+    record: "hmcts-live-ext.azurefd.net"
+  - name: "sdt"
```

Comment:
There are no corresponding A records that I can see for sdt and sdtcomm

Reply:
that's right! they both need to be taken out. Not needed at this point